src/hints/block_context.rs

```diff
@@ -137,15 +108,15 @@ pub fn load_deprecated_class(
     ids_data: &HashMap<String, HintReference>,
     ap_tracking: &ApTracking,
 ) -> Result<HintExtension, HintError> {
-    // TODO(#61): fix comp class hash
-    // let computed_hash_addr = get_ptr_from_var_name("compiled_class_fact", vm, ids_data,
-    // ap_tracking)?; let computed_hash = vm.get_integer(computed_hash_addr)?;
-    // let expected_hash = exec_scopes.get::<Felt252>("compiled_class_hash").unwrap();
-    // if computed_hash.as_ref() != &expected_hash {
-    //     return Err(HintError::AssertionFailed(
-    //         format!("Compiled_class_hash mismatch comp={computed_hash}
-    // exp={expected_hash}").into_boxed_str(),     ));
-    // }
+    let computed_hash_addr = get_ptr_from_var_name("compiled_class_fact", vm, ids_data, ap_tracking)?;
+    let computed_hash = vm.get_integer(computed_hash_addr)?;
+    let expected_hash = exec_scopes.get::<Felt252>("compiled_class_hash").unwrap();
```

Comment:
For variable names, we introduced the `vars` module and its `ids` and `scopes` submodules. Could you define constants there instead of using hardcoded strings?